### PR TITLE
Archive database backups.

### LINKED ISSUE #530: Archive database
Each time the data is published, the public database should be archived.
We need to define how the archive should be done.
Creating a dump each time the data is published doesn't seem to good solution since 95% of the data is similar from one version to an other. Instead, we should investigate the usage of `barman`.

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Fix #530

Note that developpers should have to set local ``backup_path`` setting in ``adminapp``.

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@pgiraud : Have you a precise idea about path ?
Exemple backup path : /tmp/thinkhazard.2016-04-07T15:26:32.824867.backup
This does not take care about instanceid.
Maybe we could add a setting to disable this outside of production server ?

**Comment by pgiraud** [comment]
Directory to put the archives in is `/srv/archives`. We could add a `backup` subfolder not to mix things with generated reports.
The path should be configurable (as you already did).

I would say that if the variable is not set, no backup should be done. Let's talk about the variable to documentation and remove it (or comment it) from dev/prod.ini files.

File: thinkhazard/scripts/publish.py [hunk 1]
```diff
@@ -75,6 +76,17 @@ def main(argv=sys.argv):
     admin_database = database_name(config_uri, 'admin', options=options)
     public_database = database_name(config_uri, 'public', options=options)
 
+    backup_path = os.path.join(
+        settings['backup_path'],
+        'thinkhazard.{}.backup'.format(datetime.utcnow().isoformat()))
+    print backup_path
```
Comment (pgiraud) [inline thinkhazard/scripts/publish.py]: Do be removed.

**Comment by arnaud-morvan** [comment]
I've added some outputs, and lightly refactored .ini files for lisibility (easy compare).

We need a path for backup, because we need to do create the file. A nice to have could be a setting to magically remove the backup in development mode.

**Comment by pgiraud** [comment]
LGTM. Merging.

**Pull request merged**